src/index.js

```diff
@@ -40,30 +40,46 @@ const parameters = [
   'typoTolerance'
 ]
 
-const addTypeNameToResult = (result, __typename = 'AlgoliaQuery') => Object.assign({}, result.content, { __typename })
+const prettifyResult = ({ result, type: __typename = 'AlgoliaQuery' }) =>
```

Comment:
I believe that the previous function name was more descriptive, also the previous function signature was fine for passing type as a second argument, can we please revert this change?